frontend/src/AppRoutes/index.tsx

```diff
@@ -182,6 +212,16 @@ function App(): JSX.Element {
 	}, [isLoggedInState, isOnBasicPlan, user]);
 
 	useEffect(() => {
+		if (pathname === ROUTES.ONBOARDING) {
+			window.Intercom('update', {
+				hide_default_launcher: true,
```

Comment:
should we enable this ? was wondering since we do not have skip option on the page and if user is facing some issue here then there is no way they can communicate...

Reply:
As this flow doesn't require alot of help, we choose to hide it for this to limit flows.